webapp/src/components/CollectionItemsCard.jsx

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+import CreateIcon from "./CreateIcon";
+import {
+  ICON_TYPES_LIST,
+  INNER_ICON_TYPES_LIST,
+} from "../constants/iconTypeConsts";
+import CollectionItem from "./CollectionItem";
+import { useModal } from "../hooks/modalsHooks";
+
+export default function CollectionItemsCard({ title, data }) {
+  const { showModal } = useModal();
+  const handleAddLink = () => {};
+  return (
+    <div className="collection-items-card card position-relative">
+      <div className="position-absolute" style={{ top: "32px", left: "33px" }}>
```

Comment:
Convert inline styling to class-based. Check and make changes for all the usages within this file.